common/device-model.cpp

```diff
@@ -881,7 +881,10 @@ namespace rs2
                 auto advanced = dev.as<advanced_mode>();
                 if (advanced.is_enabled())
                 {
-                    draw_advanced_mode_controls(advanced, amc, get_curr_advanced_controls, was_set, error_message);
+                    std::string dev_name = dev.supports(RS2_CAMERA_INFO_NAME) ? dev.get_info(RS2_CAMERA_INFO_NAME) : "";
```

Comment:
We also have a D430 GMSL camera.
Better we change to
```
 if( dev.supports( RS2_CAMERA_INFO_CONNECTION_TYPE ) )
 {
     auto con_type = std::string( dev.get_info( RS2_CAMERA_INFO_CONNECTION_TYPE ) );
     if( con_type == "GMSL" )
         host_assistance = 1;  // To be used for MIPI SKU only
 }
```

Please make sure the info registered before this lines, if not we will probably have a `mipi_device` variable or something

Reply:
Fixed

Reply:
Is D430 GMSL using new AE algorithm?
I think it's not implemented for D435 family.

Reply:
You are right, good catch.
@noacoohen  let's check in the office this week if D430 GMSL supports this feature before we disable it.
Please schedule us some time  to test it together.

Reply:
As it seems teoretically, it should only be blocked by D457 some I probably misled you, sorry :)

Reply:
As we tested this should be blocked only on D457